models/intel/text-to-speech-en-0002/text-to-speech-en-0001-decoder/model.yml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2022 Intel Corporation
```

Comment:
probably these files will be automatically generated, please remove

Reply:
also there is misprint in path: models/intel/text-to-speech-en-0002/text-to-speech-en-0001-decoder/model.yml -> models/intel/text-to-speech-en-0002/text-to-speech-en-0002-decoder/model.yml